### PR TITLE
week 5 Project-guess-who 

### PR DESCRIPTION
**Pull request by tuttibalutti**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -240,104 +244,156 @@ const generateBoard = () => {
...
+  charactersInPlay = characters;
+  generateBoard();
+  setSecret();
+  questions.value = questions.options[0];
+  board.style.display = "flex";
+  winOrLoseSection.style.display = "none";
+};
 
-// setting the currentQuestion object when you select something in the dropdown
+// Function that sets the personFeature object when you select something in the dropdown
```
Comment (FD000) [inline script.js:267]: Very good description of what this Function does :)

File: script.js [hunk 2]
```diff
@@ -240,104 +244,156 @@ const generateBoard = () => {
...
+};
 
-// It'll filter the characters array and redraw the game board.
+// Function that filter characters array and redraw the game board.
 const filterCharacters = (keep) => {
-  // Show the correct alert message for different categories
-  if (group === 'accessories') {
+  let group = personFeature.category;
+
+  // Alert messages for different categories
```
Comment (FD000) [inline script.js:310]: Well done :)

**Review by FD000 — COMMENTED** [review]
Well done 🥇  you have done a good work and everything looks great. What you could have done differently is to maybe change the background color in css in order to just practice. The more you change the features, the more you will practice, and more practice is always good 👍

File: index.html [hunk 3]
```diff
@@ -26,14 +26,26 @@ <h1>Does the person have</h1>
         <optgroup label="hair color">
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
+          <option value="black">black hair</option>
+          <option value="orange">orange hair</option>
+          <option value="purple">purple hair</option>
+          <option value="white">white hair</option>
+          <option value="grey">grey hair</option>
+          <option value="hidden hair">hidden hair</option>
           <!-- Add the rest of hair colors as options  -->
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply index.html:35]: Yes, remember to always clean your code from all console.log()s and commented out code blocks and tips left for you by teachers 🗡️ ✂️

File: script.js [hunk 4]
```diff
@@ -240,104 +244,156 @@ const generateBoard = () => {
...
 
-// setting the currentQuestion object when you select something in the dropdown
+// Function that sets the personFeature object when you select something in the dropdown
 const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
-  // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
+  const selectedCategory = questions.options[questions.selectedIndex].parentNode.label;
+  const selectedValue = questions.options[questions.selectedIndex].value;
+  console.log(selectedValue, selectedCategory);
```
Comment (KarolCislowski) [inline script.js:271]: Remember to remove all console.log()s

**Review by KarolCislowski — APPROVED** [review]
This is a very nice project 🌈 

I know it was an extra point of this assignment to style it your way but remember that there is no another way to master CSS then practice it over and over again 🦾 

Remember that you can always come back to it when you have some spare time and add more functionalities to make it an even greater portfolio asset! 🦄

**Pull request closed without merging**